js/angular-crmMailing/directives.js

```diff
@@ -28,6 +28,7 @@
           });
           scope.crmMailingConst = CRM.crmMailing;
           scope.ts = CRM.ts(null);
+          scope.helpFile = 'CRM/Mailing/Form/Upload';
```

Comment:
Proposals (for sake of discussion):
 * templates/CRM/Mailing/crmMailing.hlp
 * templates/CRM/Mailing/Help.hlp
 * templates/CRM/Mailing.hlp
 * templates/angular/crmMailing.hlp

Aside: I'm still pretty tempted to rearrange the files for the Angular components, e.g.
  * templates/crmMailing/module.js
  * templates/crmMailing/controllers.js
  * templates/crmMailing/directives.js
  * templates/crmMailing/partials/edit.html
  * templates/crmMailing/help.hlp